pom.xml

```diff
@@ -1,9 +1,86 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.openmrs</groupId>
-  <artifactId>openmrs-content-hiv</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <name>Archetype - openmrs-content-hiv</name>
-  <url>http://maven.apache.org</url>
+	<modelVersion>4.0.0</modelVersion>
+ 	<groupId>org.openmrs.content</groupId>
+  	<artifactId>openmrs-content-hiv</artifactId>
+  	<version>1.0-SNAPSHOT</version>
+  	<name>OpenMRS HIV Content Package</name>		
+	<packaging>pom</packaging>
+	<developers>
+		<developer>
+			<name>OpenMRS</name>
+			<organization>OpenMRS</organization>
+			<organizationUrl>http://openmrs.org</organizationUrl>
+		</developer>
+	</developers>
+	<scm>
+		<connection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</connection>
+		<developerConnection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</developerConnection>
+		<url>https://github.com/openmrs/openmrs-content-hiv.git</url>
+		<tag>HEAD</tag>
+	</scm>	
+	<dependencies>
+		<dependency>
```

Comment:
With the OpenMRS code formatter, tabs are used for indenting and each is equals 4 spaces. Ideally, commits should be made after running `mvn clean install` on projects utilising the OpenMRS formatter, so that formatting changes can get in too.

Reply:
Done, Thank you!